default_payment_config.xml

```diff
@@ -108,6 +126,23 @@
       <wdoxidee_descriptor>1</wdoxidee_descriptor>
       <wdoxidee_additional_info>1</wdoxidee_additional_info>
     </payment>
+    <payment>
```

Comment:
Please remove this payment config as it is a **duplicate** from above.

Reply:
:+1: